### ISSUE TITLE
More controlled values for featureType

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/camtrap-dp#371]
_Originally posted by @vechocho in https://github.com/gbif/ipt/issues/2209#issuecomment-1821395994_

> @peterdesmet on the [featureType](https://camtrap-dp.tdwg.org/data/#deployments.featureType) as it is not a mandatory field, maybe consider using `other` or let people use their own vocabulary or should I open a issue on the camtrap github? As i see now all of the features have relation with roads or places with easy access. On our deployment we put the cameras on places where we see animal crossing, or they go for salt. sometimes on the mountains on the andes we dont have trees... So i see the vocabulary is more made for forest type ecosystems and not for deserts or andean types. On our case the cameras are plced by ranger that go on places not open on the protected areas for more people.

**Comment by peterdesmet** [comment]
@vechocho, quick answers:

1. We decided to remove `other` from controlled vocabularies (see https://github.com/tdwg/camtrap-dp/issues/251), because it defeats the purpose of a controlled vocabulary a bit. Rather, we want people to suggest more values for a controlled vocabulary, so it becomes better overall. This is exactly what you are doing here.
2. The featureType vocabulary was borrowed from [CTMS](https://doi.org/10.3897%2FBDJ.4.e10197), but I agree it has a limited scope. **What values would you suggest to add to the vocabulary?**
3. Note that presence of salt can be indicated as `baitUse=TRUE`. This doesn't specify the type of bait though, for which you have to rely on tags or comments. I'll let others weigh in on whether salt is a type of feature.

**Comment by peterdesmet** [comment]
@vechocho do you have suggestion for new values that should be added to [featureType](https://camtrap-dp.tdwg.org/data/#deployments.featureType) or is your use case solved?

**Comment by tomaugust** [comment]
Yes please - I have need for new values here for bat recordings. In particular I want to know if the recording was made at a 'roost' - the location where bats rest/sleep; 'swarming site' - a place where bats congregate to mate; 'free flight' - A bat out and about. Now, that latter is a bit undefined and feel a bit 'other', as such perhaps what I really need is hte other two, and then a bat that is out and about is simply not asscoiated with a feature. But @peterdesmet how could I tell between a recording that was 'out and about' versus a recording that was at a roost, but the field had not been filled in??
